docs/content/docs/Infrastructure.md

```diff
@@ -175,4 +175,4 @@ You will see branches from both repositories:
 - By default, build uses develop from github against develop from avito
 - If you pick a branch from avito, it will run against develop on github
 - If you pick a branch from github, it will run against develop on avito
-- (UNTESTED) To build both projects of special branch, they should have the same name
```

Comment:
I've tested it. Works as expected.

Reply:
The same

Reply:
I see only one inconvenience If I want to check backward compatibility also.

There are at least two options:
- Remove branch from avito repository
- Make a new branch from feature-branch in github

Can you please add this case to the documentation?

Reply:
I don't think that this information is somehow related to this document. The doc only describes how to run infra against Avito. It doesn't describe the process itself or how to perform custom checks.

Reply:
> The doc only describes how to run infra against Avito

Not only. This doc describes publishing and testing changes: **Publishing a new release > CI integration tests against avito**
Now we have